### ISSUE TITLE
Migration file missing

### ISSUE THREAD

**Issue by srikanth148** [celery/django-celery-beat#90]
Hi ,
Please create solarschedule table migration file.

**Comment by picturedots** [comment]
I would recommend to everyone **not** to migrate to version 1.1.0 and wait for the next release with the migration added.  

Should be something like:

```# -*- coding: utf-8 -*-
# Generated by Django 1.11.6 on 2017-10-31 19:36
from __future__ import unicode_literals

from django.db import migrations, models


class Migration(migrations.Migration):

    dependencies = [
        ('django_celery_beat', '0004_auto_20170221_0000'),
    ]

    operations = [
        migrations.AlterField(
            model_name='solarschedule',
            name='event',
            field=models.CharField(choices=[('dawn_civil', 'dawn_civil'), ('dawn_nautical', 'dawn_nautical'), ('sunset', 'sunset'), ('solar_noon', 'solar_noon'), ('sunrise', 'sunrise'), ('dusk_astronomical', 'dusk_astronomical'), ('dawn_astr
onomical', 'dawn_astronomical'), ('dusk_civil', 'dusk_civil'), ('dusk_nautical', 'dusk_nautical')], max_length=24, verbose_name='event'),
        ),
    ]
```

**Comment by ShaheedHaque** [comment]
I believe this issue can be closed, as the fix was in #92.

**Comment by mbaechtold** [comment]
I think this has been fixed in 1.1.1.
